test/TestUtilities/Test.Utility/SourceRepository/TestSourceRepositoryUtility.cs

```diff
@@ -52,6 +53,18 @@ public static SourceRepositoryProvider CreateSourceRepositoryProvider(IEnumerabl
             return sourceRepositoryProvider;
         }
 
+        public static SourceRepositoryProvider CreateSourceRepositoryProvider(IEnumerable<PackageSource> packageSources, IEnumerable<Lazy<INuGetResourceProvider>> mockNuGetResourceProviders)
+        {
+            var thisUtility = new TestSourceRepositoryUtility();
+            //var container = thisUtility.Initialize();
```

Comment:
no commented out code please

Reply:
Already did that but didn't commit. 

Fixed it again.